tool/tbot/main.go

```diff
@@ -396,9 +396,16 @@ func onMigrate(
 		out = f
 	}
 
+	var cfg *config.BotConfig
+	var err error
 	// We do not want to load an existing configuration file as this will cause
 	// it to be merged with the provided flags and defaults.
-	cfg, err := config.ReadConfigFromFile(globalCfg.ConfigPath, true)
+	if globalCfg.ConfigString != "" {
```

Comment:
We probably ought to have some validation here that detects if both have been specified, and returns an error indicating their mutually exclusive nature.